packages/web/src/pages/sign-up-page/pages/SelectGenrePage.tsx

```diff
@@ -52,40 +53,76 @@ export const SelectGenrePage = () => {
     [dispatch, navigate]
   )
 
+  const { isMobile } = useMedia()
+
   return (
-    <div>
-      <div>
-        <p>{displayName}</p>
-        <p>{handle}</p>
-      </div>
-      <h1>{messages.header}</h1>
-      <p>{messages.description}</p>
+    <Flex direction='column' gap={isMobile ? '2xl' : '4xl'}>
+      <AccountHeader />
+      <Flex direction='column' gap={isMobile ? 'xl' : '2xl'}>
+        <Flex direction='column' gap={isMobile ? 's' : 'l'} ph='l'>
+          <Text
+            variant='heading'
+            size={isMobile ? 'm' : 'l'}
+            color='heading'
+            css={{ textAlign: isMobile ? 'left' : 'center' }}
+          >
+            {messages.header}
+          </Text>
+          <Text
+            variant='body'
+            size={isMobile ? 'm' : 'l'}
+            css={{ textAlign: isMobile ? 'left' : 'center' }}
+          >
+            {messages.description}
+          </Text>
+        </Flex>
 
-      <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-        {({ values, setValues }) => {
-          const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
-            setValues({ ...values, [e.target.name]: !values[e.target.name] })
-          }
+        <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+          {({ values, setValues }) => {
+            const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+              setValues({ ...values, [e.target.name]: !values[e.target.name] })
+            }
 
-          return (
-            <Form>
-              {genres.map((genre) => {
-                const { label, value } = genre
-                return (
-                  <GenrePill
-                    key={value}
-                    label={label}
-                    name={value}
-                    checked={!!values[value]}
-                    onChange={handleChange}
-                  />
-                )
-              })}
-              <Button type='submit'>{messages.continue}</Button>
-            </Form>
-          )
-        }}
-      </Formik>
-    </div>
+            return (
+              <Flex
+                as={Form}
+                alignItems={isMobile ? 'flex-start' : 'center'}
+                direction='column'
+              >
+                <Flex
+                  css={{ maxWidth: 608 }}
```

Comment:
feels like we have some arbitrary numbers floating around for sizing. Probably copied from the designs? May need a consistency pass once things are done, and maybe some shared constants

Reply:
yeah super random agreed... what should we do about that? probably set these at higher level across multiple screens?

Reply:
yep that sounds right. That's why I think we can circle back after we've done this

Reply:
Agreed! Would love to chat / learn more about the best way to use our harmony sizing when mocking large screens as well as best practice for setting the margin per device size. 

In signup, I really wanted to limit how wide we allow the design to stretch on step 1 and 2 to increase legibility and give the experience more breathing room. But for selecting artists, we really needed that screen space. Happy to chat about the cleanest solution here and update the Figma to match our decision